frontend/storage/utils/createGameAbilitiesListData.ts

```diff
@@ -0,0 +1,38 @@
+import { AbilityExample } from '../models/AbilityExample';
+import { Ability } from '../models/Ability';
+import { AbilityGroup } from '../models/AbilityGroup';
+import { AbilityCategory } from '../models/AbilityCategory';
+import { GameAbilitiesListProps } from '../../components/GameAbilitiesList';
+
+interface GameAbilitiesListInitialProps {
+	abilityExamples: Array<AbilityExample>;
+	abilities: Map<number, Ability>;
+	abilityGroups: Map<number, AbilityGroup>;
+	abilityCategories: Map<number, AbilityCategory>;
+	gameId: number;
+}
+export function createGameAbilitiesListData(props: GameAbilitiesListInitialProps): GameAbilitiesListProps {
```

Comment:
at the start of the function do: `const {abilityExamples, abilities...} = props` to avoid the constant references to `props` which make things a bit less readable.